utils/input.py

```diff
@@ -142,8 +142,8 @@ def to_watcher(watcher, hotkey):
 
         if is_keyboard_module_available or is_pyinput_module_available:
             self.hotkey_watcher.start()
-
-        self.clipboard_watcher.start()
+        else:
```

Comment:
You are still removing the ctrl+c functionality with this. It would be nice if I did not had to repeat myself for like 4th time in a row with this

Reply:
ctrl+c functionality will need to be restored before I will merge this. Also, please implement deathbeam's other comments concerning multiple components. I for one would not enjoy having a large chunk of the screen covered in GUI when bulk pricing items.

Reply:
If i dont have the else statement there it calls the item search twice when i press alt+d, You said earlier that you wanted the clipboard_watcher as a backup if the other one failed thats why I added else: self.clipboard_watcher.start().

Reply:
But I will look for a way to prevent it in some other way..

Reply:
How did you managed to regress into that state from what is in master? As on master even with the clipboard watcher it happens only once. I guess its because of the deferring of hotkey callbacks back to main thread, so its actually not blocking hotkey watcher what is atm preventing double presses.

And yeah, primary clipboard watcher purpose is to at least do something when pynput and keyboard are unavailable, but it is also used as primary handler for ctrl+c atm, without it there is simply no ctrl-c.

Also something that I did not mentioned before, check what keyboard watcher is doing atm. It makes sense on master, but not on your PR. because what is atm happening in your PR is that  every hotkey is sent to hotkey watcher for processing (so it can be handled in async manner, at least that was the plan) but with your queue you immediatelly send it back to main thread.